aten/src/ATen/native/SpectralOpsUtils.h

```diff
@@ -3,8 +3,14 @@
 #include <string>
 #include <stdexcept>
 #include <sstream>
+#include <c10/core/ScalarType.h>
+#include <c10/util/ArrayRef.h>
+#include <c10/util/Exception.h>
 #include <ATen/native/DispatchStub.h>
 
+namespace at {
+class Tensor;
```

Comment:
Why do you need that?

Reply:
As linter wants to include SpectralOpsUtils.h first on the list, and want to fwd declare tensor. But can replace it with `#include` if needed

Reply:
If including TensorBase is enough I would.
Otherwise the forward declaration is fine yep